.github/workflows/run-tests-on-pr.yaml

```diff
@@ -0,0 +1,13 @@
+on:
+  pull_request_review:
+    types: [submitted]
+
+jobs:
+  testing-approved-pr:
+    if: github.event.review.state == 'APPROVED'
+    uses: precice/tutorials/.github/workflows/run_testsuite_workflow.yml@develop
+    with:
+      suites: openfoam_adapter_pr
+      build_args: TUTORIALS_REF:develop,PRECICE_REF:develop,OPENFOAM_EXECUTABLE:openfoam2306,OPENFOAM_ADAPTER_REF:${{ github.event.pull_request.head.sha }}
```

Comment:
Now that we only support commits and tags, we need a way to specify that we need the latest develop branches.

Reply:
This currently gives:

```
build_args=TUTORIALS_REF:develop,PRECICE_REF:develop,OPENFOAM_EXECUTABLE:openfoam2306,OPENFOAM_ADAPTER_REF:7d19f75264028f575e44d980f7e91715f3d21222 --suites=openfoam_adapter_pr --log-level=DEBUG
```

The adapter is correct, but the develop branches of the other components are probably not the very latest. Maybe the simplest would be to have a workflow or cron job that updates/invalidates the cached develop images.

Reply:
I now used https://github.com/marketplace/actions/get-latest-commit to get the latest develop branch for the tutorials.

I had to split the job into two: one for gathering the references and one for triggering the system tests with these references.

The jobs are sharing outputs: https://docs.github.com/en/actions/using-jobs/defining-outputs-for-jobs